src/DependencyInjection/Configuration.php

```diff
@@ -65,8 +61,8 @@ public function getConfigTreeBuilder()
                             ->end()
                         ->end()
                     ->end()
-                ->arrayNode('loaders')
-                    ->info('Enabled loaders.')
+                ->arrayNode('plugins')
+                    ->info('Enabled plugins.')
```

Comment:
Are these the actual enabled plugins or the whole list of defined enabled & disabled plugins?

Reply:
This is a list of plugins, which may be enabled or not. It does not have to be complete, they will only be enabled if they are on the list & are set to true..